geometry_msgs/Point[] tracker_traj
